services/app/src/Init.tsx

```diff
@@ -95,8 +95,10 @@ function setupDevice() {
     // https://stackoverflow.com/a/43502958/1332186
     if (/Android/.test(navigator.appVersion)) {
       window.addEventListener('resize', () => {
-        if (document.activeElement.tagName === 'INPUT' || document.activeElement.tagName === 'TEXTAREA') {
-          document.activeElement.scrollIntoView();
+        if (document.activeElement !== null) {
```

Comment:
In JS, you can also abbreviate this by just checking if (document.activeElement), since if it's null, it'll be falsey :)

Reply:
Got it.